post/models.py

```diff
@@ -18,3 +17,18 @@ def get_absolute_url(self):
     # Print the string representation of Post
     def __str__(self):
         return self.post_story
+
+
+# Based on code by Django Girls, url: https://djangogirls.gitbooks.io/django-girls-tutorial-extensions/homework_create_more_models/
+class Comment(models.Model):
+    post = models.ForeignKey(Post, related_name='comments')
+    author = models.CharField(max_length=80)
```

Comment:
I assume you're referring to the form requiring a drop-down afterwards. In that case, you can probably pre-fill the that field in the form (server-side). Of course, you would then not need to send the field out to the form (view-side).